cookbook/bundles/best_practices.rst

```diff
@@ -72,49 +68,51 @@ class name.
     :class:`Symfony\\Bundle\\FrameworkBundle\\FrameworkBundle`.
 
 Each bundle has an alias, which is the lower-cased short version of the bundle
-name using underscores (``acme_hello`` for ``AcmeHelloBundle``, or
-``acme_social_blog`` for ``Acme\Social\BlogBundle`` for instance). This alias
-is used to enforce uniqueness within a bundle (see below for some usage
-examples).
+name using underscores (``acme_blog`` for ``AcmeBlogBundle``). This alias
+is used to enforce uniqueness within a project and for defining bundle's
+configuration options (see below for some usage examples).
 
 Directory Structure
 -------------------
 
-The basic directory structure of a HelloBundle must read as follows:
+The basic directory structure of an AcmeBlogBundle must read as follows (if you
+use `PSR-4`_ the ``Acme/`` and ``BlogBundle/`` folders might not exist):
```

Comment:
might not exists could confuse (because the content exists)? any ideas for a better wording here?